lib/c-api/wasmer_wasm.h

```diff
@@ -29,9 +29,6 @@
 #  define DEPRECATED(message) __declspec(deprecated(message))
 #endif
 
-// The `jit` feature has been enabled for this build.
-#define WASMER_JIT_ENABLED
-
```

Comment:
??

Reply:
Oops, not sure how that ended up in there